File: kotlin/vci-client/src/test/java/io/mosip/vciclient/common/UtilTest.kt
package io.mosip.vciclient.common

import org.junit.Assert.*
import org.junit.Test

class UtilTest {
    @Test
    fun `should return log tag with library name and provided class name`() {
        val traceabilityId = "test-vci-client"
        val logTag: String = Util.getLogTag(javaClass.simpleName, "$traceabilityId")

        assertEquals("INJI-VCI-Client : ${javaClass.simpleName} | traceID $traceabilityId", logTag)
    }
}

File: js/test/common/Util.test.js
const Logger =  require("../../src/common/Logger");

test('should set and return log tag with library name and provided class name', () => {
  
  Logger.setLogTag("AX123","UtilTest");
  
  const logTag = Logger.getLogTag();
  
  expect(logTag).toBe('INJI-VCI-Client: UtilTest | traceID: AX123 |');
});
